_build.ts

```diff
@@ -63,7 +62,7 @@ await emptyDir("./target/wasm32-bindgen-deno-js");
 
 const wasmBindGenCmd = [
   "wasm-bindgen",
-  "./target/wasm32-unknown-unknown/release/deno_emit.wasm",
+  "./target/wasm32-unknown-unknown/release/emit.wasm",
```

Comment:
It wouldn't let me build `deno_emit_wasm` as `deno_emit` because of the `deno_emit` crate... so `emit` it is.